modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/user/application/v1/UserDiscoverableApplicationServiceTestBase.java

```diff
@@ -95,6 +97,8 @@ private void createServiceProviders() throws Exception {
             serviceProvider = appMgtclient.getApplication(APP_NAME_PREFIX + i);
             if (serviceProvider != null) {
                 serviceProvider.setDiscoverable(true);
+                serviceProvider.setImageUrl(APP_IMAGE_URL);
```

Comment:
@malithie @darshanasbg 

These dummy URLs are used as the service providers are set as discoverable. I assumed that we are not testing the flows where image URLs and access URLs used.

Reply:
We have to validate the set URLs in API response path as well

Reply:
Addressed in a52a880ae74fc40dd0480e1eb5f48c64eb1ea79c.